imx-bootlets: Fix paths for used during boot stream generation for Austin board

--- a/updater.bd
+++ a/updater.bd
@@ -4,11 +4,11 @@
 	flags = 0x01;
 }
 sources {
-	power_prep="./power_prep/power_prep";
-	sdram_prep="./boot_prep/boot_prep";
-	linux_prep="./linux_prep/output-target/linux_prep";
-	zImage="zImage";
-	initramfs="initramfs.cpio.gz";
+	power_prep="imx-bootlets-power_prep-@MACHINE@";
+	sdram_prep="imx-bootlets-boot_prep-@MACHINE@";
+	linux_prep="imx-bootlets-linux_prep-@MACHINE@";
+	zImage="zImage-@MACHINE@.bin@DTB@";
+	initramfs="austin-initramfs-image-@MACHINE@.cpio.gz";
 }
 
 section (0) {
